src/components/GlobalSearch.vue

```diff
@@ -0,0 +1,72 @@
+<!-- 全局搜索组件 -->
```

Comment:
组件注释可以写到 script 下，可以采用这种注释比较清晰

```suggestion
/**
 * @file xx
 */
```

Reply:
fix